module/dkg/utils.go

```diff
@@ -0,0 +1,22 @@
+package dkg
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func GetDKGCommitteeIndex(nodeID flow.Identifier, participants flow.IdentityList) (int, error) {
+	myIndex := -1
+	for i, id := range participants.NodeIDs() {
+		if id == nodeID {
+			myIndex = i
+			break
+		}
+	}
+	if myIndex < 0 {
+		return myIndex, fmt.Errorf("node does not belong to dkg committee")
```

Comment:
if we use an error, better to include the nodeID in it

Reply:
https://github.com/onflow/flow-go/pull/1849/commits/aa369fc7718a37db5db1e76cb77481bc0c1cca33